Runtime/Scripts/Read3DTileset.cs

```diff
@@ -105,15 +115,13 @@ public void ConstructURLWithKey()
             //Append the key query parameter
 #if UNITY_EDITOR
             if (!string.IsNullOrEmpty(personalKey))
-            {
-                uriBuilder.Query += $"{QueryKeyName}={personalKey}";
-            }
 #else
             if (!string.IsNullOrEmpty(publicKey))
+#endif
```

Comment:
Moving this endif will always trigger the code below, and thus overwriting the personalKey setting in the UNITY_EDITOR. Is this an expected change?

Reply:
no, I just moved it so that only the if is preprocessed, since the body of the if statement was the same for both editor and non-editor versions

Reply:
Ah yes! I misread the line! Nothing to see here ;) 👌